docs/k0sctl-install.md

```diff
@@ -0,0 +1,177 @@
+
+# Deploying a k0s cluster using k0sctl
+
+k0sctl is a command-line tool for bootstrapping and management of k0s clusters. Installation instructions can be found in the [k0sctl github repository](https://github.com/k0sproject/k0sctl#installation).
+
+k0sctl will connect to provided host using ssh and gather information about the host. Based on the finding it will proceed to configure the host in question and install k0s binary.  
+## Using k0sctl
+First create a k0sctl configuration file:
+```sh
+$ k0sctl init > k0sctl.yaml
+```
+
+A `k0sctl.yaml` file will be created in the current directory:
+
+```yaml
+apiVersion: k0sctl.k0sproject.io/v1beta1
+kind: Cluster
+metadata:
+  name: k0s-cluster
+spec:
+  hosts:
+  - role: controller
+    ssh:
+      address: 10.0.0.1 # replace with the controller's IP address
+      user: root
+      keyPath: ~/.ssh/id_rsa
+  - role: worker
+    ssh:
+      address: 10.0.0.2 # replace with the worker's IP address
+      user: root
+      keyPath: ~/.ssh/id_rsa
+  k0s:
+    version: v0.11.0-beta.2
```

Comment:
```suggestion
```

```suggestion
    version: v0.11.0
```

I don't think the installation example should install an old version.

If you remove the whole `spec.k0s.version`, it will discover the latest version, so, the example should install the latest, even if new versions are released, as long as the rest of the api is still compatible.